src/main/java/LocalKeyClientEncryption.java

```diff
@@ -0,0 +1,158 @@
+    }
+
+    /**
+     * Creates a random, not secure local key to be used in client-side encryption
+     */
+    private static AsyncKeyEncryptionKey createLocalKey(){
+        byte[] byteKey = new byte[32];
+        new Random().nextBytes(byteKey);
+        JsonWebKey localKey = JsonWebKey.fromAes(new SecretKeySpec(byteKey, "AES"),
+                Arrays.asList(KeyOperation.WRAP_KEY, KeyOperation.UNWRAP_KEY))
+                .setId("my-id");
+        AsyncKeyEncryptionKey akek = new LocalKeyEncryptionKeyClientBuilder()
+                .buildAsyncKeyEncryptionKey(localKey).block();
+        return akek;
+    }
+
+    /**
+     * Sets local key client with in helper object to be used in decrpytion
+     */
+    private static EncryptedBlobClient createLocalKeyClient(AsyncKeyEncryptionKey key, BlobClient blobClient){
+        return new EncryptedBlobClientBuilder()
+                .key(key, KeyWrapAlgorithm.A256KW.toString())
+                .blobClient(blobClient)
+                .buildEncryptedBlobClient();
+    }
+
+    /**
+     * Creates example container and blob, then uploads with client-side encryption with local keys
+     */
+    public static BlobClient setup(String storageAccount, String sharedKeyCred, String containerName, String blobName,
+                                   String blobSuffix, AsyncKeyEncryptionKey key){
+        String storageAccountUrl = "https://" + storageAccount + ".blob.core.windows.net";
+
+        // Creating a BlobServiceClient that allows us to perform container and blob operations, given our storage
+        // account URL and shared key credential
+        BlobServiceClient blobServiceClient = new BlobServiceClientBuilder()
+                .endpoint(storageAccountUrl)
+                .credential(new StorageSharedKeyCredential(storageAccount, sharedKeyCred))
+                .buildClient();
+
+        // Creating client referencing to-be-created container, and then creating it
+        BlobContainerClient blobContainerClient = blobServiceClient.getBlobContainerClient(containerName);
+        blobContainerClient.create();
+
+        // Creating a blob client
+        BlobClient blobClient = blobContainerClient.getBlobClient(blobName + blobSuffix);
+
+        // Setting encryptedKeyClient
+        EncryptedBlobClient encryptedBlobClient = createLocalKeyClient(key, blobClient);
+
+        // Uploading example blob with client-side encryption
+        String fileName = blobName + blobSuffix;
+        encryptedBlobClient.uploadFromFile(".\\src\\main\\java\\" + fileName);
+        return blobClient;
+    }
+
+    /**
+     * Downloads client-side encrypted blob, decrypts with local key, then reuploads with server-side encryption
+     */
+    public static void decryptReupload(BlobClient blobClient, String storageAccount, String sharedKeyCred, String containerName, String blobName,
+                                String blobSuffix, AsyncKeyEncryptionKey key, CustomerProvidedKey serverKey){
```

Comment:
nit: make sure you have spaces between end of the method def and the curly bracket